README.md

```diff
@@ -403,11 +422,11 @@ For each element, with weight `w`, meeting at least one of the above requirement
 5) `d1` is computed, such that:
```

Comment:
☝️ On line 400 of the original:

```
                       |
                       V
1) `d0 = 1`, if `w1 <= = && w1 + w2 > 0`
```

I'm guessing that this is the typo, not the `<=`?